### PR TITLE
New profile: session-desktop

### PR DESCRIPTION
**Pull request by glitsj16**
session-desktop

Description: Encrypted messenger.

https://github.com/oxen-io/session-desktop/
https://aur.archlinux.org/packages/session-desktop
https://aur.archlinux.org/packages/session-desktop-bin
https://aur.archlinux.org/packages/session-desktop-appimage

Note: The AUR packages all work with the profiles.

### CODE REVIEW

**Comment by glitsj16** [comment]
> What do you mean, why does it affect adding them to firecfg.config?

You're correct, the differences don't affect firecfg. Added!

File: etc/session-desktop.profile [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+noroot
+notv
+nou2f
+protocol unix,inet,inet6,netlink
+seccomp !chroot
+seccomp.block-secondary
+tracelog
+
+disable-mnt
+private-bin bash,cat,dirname,echo,env,exec,session-desktop*,session-messenger-desktop*,sh
```
[the remark this answers is not in the document]
Comment (glitsj16) [reply etc/session-desktop.profile]: With the latest versions there are no wrapper scripts. Corrected that just now. Only difference is packaging (in the AUR) are the symlinked launchers.

File: etc/session-desktop.profile [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+private-cache
+?HAS_APPIMAGE: ignore private-dev
+private-dev
+private-etc @network,@tls-ca,@x11
+private-opt Session,session-desktop
```
[the remark this answers is not in the document]
Comment (glitsj16) [reply etc/session-desktop.profile:57]: Not sure what you mean. There's the usual whitelisting in session-desktop.profile. Isn't that what you're referring to?

[the remark this answers is not in the document]
Comment (kmk3) [reply etc/session-desktop.profile:57]: > Not sure what you mean. There's the usual whitelisting in
> session-desktop.profile. Isn't that what you're referring to?

I mean `whitelist /opt/...` as in #6021, to reduce RAM usage.

[the remark this answers is not in the document]
Comment (glitsj16) [reply etc/session-desktop.profile]: Oh I see, now I get it. Done!

**Review by kmk3 — COMMENTED** [review]
* etc/profile-m-z/session-messenger-desktop.profile
* etc/profile-m-z/session-messenger.profile

What is each redirect for?

Are they for wrapper scripts, AppImages, or just different binary names on
different distributions?

Why are they not also added to firecfg.config?

**Comment by glitsj16** [comment]
> Are they for wrapper scripts, AppImages, or just different binary names on
different distributions?

The latter. Even on the same distribution (Arch Linux) this app can have differently symlinked binary names in /usr/bin. I've added the redirects to firecfg. Thanks for the review!

**Review by kmk3 — APPROVED** [review]

**Pull request merged**